services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/core.py

```diff
@@ -155,12 +156,14 @@ async def _retrieve_output_data(
                 exc=exc,
             ) from exc
 
-    async def _publish_sidecar_log(self, log: str) -> None:
+    async def _publish_sidecar_log(
+        self, log: str, log_level: int = logging.INFO
```

Comment:
``log`` here is the ``message`` log , right?

Reply:
yes